api/v1/strict_server.go

```diff
@@ -298,6 +298,14 @@ func (srv *StrictServerImpl) GetRuntimeEvmTokensAddressNfts(ctx context.Context,
 	return apiTypes.GetRuntimeEvmTokensAddressNfts200JSONResponse(*nfts), nil
 }
 
+func (srv *StrictServerImpl) GetRuntimeEvmTokensAddressNftsId(ctx context.Context, request apiTypes.GetRuntimeEvmTokensAddressNftsIdRequestObject) (apiTypes.GetRuntimeEvmTokensAddressNftsIdResponseObject, error) {
+	nft, err := srv.dbClient.RuntimeEVMNFT(ctx, request.Address, request.Id)
```

Comment:
To reduce boilerplate and copy-paste, I'd prefer if we reuse `srv.dbClient.RuntimeEVMNFTs()` here (note the plural in the name). It will need to start taking a new optional param: The NFT ID. 

See `GetRuntimeEvmTokensAddress()` and `GetConsensusValidatorsEntityId()` for two places with a very similar problem, and a solution along the lines above.

Reply:
k converted